mypy_django_plugin/transformers/querysets.py

```diff
@@ -385,6 +385,9 @@ def extract_prefetch_related_annotations(ctx: MethodContext, django_context: Dja
         and ctx.args
         and ctx.arg_types
         and ctx.arg_types[0]
+        # Only process the correct overload, i.e.
+        #     def prefetch_related(self, *lookups: str | Prefetch[_PrefetchedQuerySetT, _ToAttrT]) -> Self: ...
+        and "lookups" in ctx.callee_arg_names
```

Comment:
I think it would be better to use arg kind here, because name can be changed: since it is `*args`, it is not a requirement to keep this exact name.

Reply:
yes, let's do this :)